config/initializers/sidekiq.rb

```diff
@@ -6,9 +24,15 @@
 
 Sidekiq.configure_server do |config|
   schedule_file = "config/schedule.yml"
+  # @mstruve/@sre: sidekiq-cron still uses the removed poll_interval
+  # to determine how often to poll for jobs so we should manually set it
+  # https://github.com/ondrejbartas/sidekiq-cron/issues/254
+  # Sidekiq default is 5, we don't need it quite that often but would like it more than
+  # every 30 seconds which the gem defaults to
+  Sidekiq.options[:poll_interval] = 10
 
   if File.exist?(schedule_file)
-    Sidekiq::Cron::Job.load_from_hash(YAML.load_file(schedule_file))
+    Sidekiq::Cron::Job.load_from_hash!(YAML.load_file(schedule_file))
```

Comment:
Turns out [you need a `!` here to ensure that any jobs that are removed from your yml file, also get removed from Redis.](https://github.com/ondrejbartas/sidekiq-cron/issues/267#issuecomment-573496747)